www/js/draw-model.js

```diff
@@ -502,9 +503,20 @@ define([
 					this._equationNodeCount++;
 				}
 				this.addNodeCount();
+				if(type==="quantity") this.sortQuantityDescriptions();
 			}
 		},
-
+		sortQuantityDescriptions: function(){
+			var quantityParent = dom.byId("quantity-description");
+			var quantityDivs = quantityParent.querySelectorAll(".quantityDescription");
+			var sortedArr = Array.prototype.slice.apply(quantityDivs);
+			sortedArr.sort(function(a,b){
+				return dom.byId(a).querySelector(".descriptionText b").innerHTML.toLowerCase() >  dom.byId(b).querySelector(".descriptionText b").innerHTML.toLowerCase();
```

Comment:
Please restructure the code and write the code at appropriate places. The logic for sorting the descriptions should not be written in draw-model. The idea here is that if we write it in model.js the code will be reusable. Else if we want to use this sorted object somewhere, we can not do that as we will need to make draw-model object somewhere else, which defeats the purpose of the whole thing.